src/drc/api/viewsets.py

```diff
@@ -95,10 +101,42 @@ class EnkelvoudigInformatieObjectViewSet(NotificationViewSetMixin,
         'destroy': SCOPE_DOCUMENTEN_ALLES_VERWIJDEREN,
         'update': SCOPE_DOCUMENTEN_BIJWERKEN,
         'partial_update': SCOPE_DOCUMENTEN_BIJWERKEN,
+        'lock': SCOPE_DOCUMENTEN_LOCK,
+        'unlock': SCOPE_DOCUMENTEN_LOCK | SCOPE_DOCUMENTEN_GEFORCEERD_UNLOCK
     }
     notifications_kanaal = KANAAL_DOCUMENTEN
     audit = AUDIT_DRC
 
+    @swagger_auto_schema(request_body=LockEnkelvoudigInformatieObjectSerializer,
+                         responses={200: LockEnkelvoudigInformatieObjectSerializer})
```

Comment:
see the `download` action to add the rest of the basic set of error responses. Especially the 400 with `ValidatieFoutSerializer` is important

Reply:
error responses are added